src/js/app.js

```diff
@@ -0,0 +1,39 @@
+import { timeDifference, timeDifferenceFuture } from './utils'
+
+export default function init() {
+  document.getElementById('ago').textContent = `(${timeDifference(
+    new Date().getTime(),
+    new Date(document.getElementById('time').dataset.time).getTime()
+  )})`
+
+  // select all leaderboards
+  var allLeaders = document.querySelectorAll('div.org ul')
+  allLeaders.forEach(function(orgList) {
```

Comment:
Is there a meaning to use `function(orgList) {` instead of `orgList => {`? If not, you are not consistent with your code since in another file it use arrow function.